AveryLabels.py

```diff
@@ -24,18 +21,21 @@
 # page margins left/top
 
 labelInfo = {
+    # 22x 32mm x 10mm mini labels
+    3044: ( 2, 11, (32, 10), (2,2), (1, 1), (66.5*mm, 120.5*mm)),
+
     # 2.6 x 1 address labels
-    5160: ( 3, 10, (187,  72), (11, 0), (14, 36)),
-    5161: ( 2, 10, (288,  72), (0, 0), (18, 36)),
+    5160: ( 3, 10, (187,  72), (11, 0), (14, 36), LETTER),
```

Comment:
Actually my label sheets are `A4` size, not `LETTER`. So let's use that.

Reply:
I used `LETTER`, since this was the default (`self.pagesize = LETTER`) until now. Replaced it all with `A4` now.

Reply:
Yeah, I realized that as well after you copied it to the label definitions. I blindly copied from the original at https://gist.github.com/timrprobocom/3946aca8ab75df8267bbf892a427a1b7#file-averylabels-py-L52
Thanks for the change.